### PR TITLE
Impact of features on the accuracy of an AD model

### PR DESCRIPTION
**Pull request by ashwinkumar12345**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by aetter — APPROVED** [review]

File: docs/ad/index.md [hunk 1]
```diff
@@ -52,7 +52,10 @@ In this case, a feature is the field in your index that you to check for anomali
 
 For example, if you choose `min()`, the detector focuses on finding anomalies based on the minimum values of your feature. If you choose `average()`, the detector finds anomalies based on the average values of your feature.
 
-You can add a maximum of five features for a detector.
+A multi-feature model correlates anomalies across all its features. It's difficult for the model to identify anomalies for an individual feature. Adding more features might negatively impact the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) of a model.
```
Comment (kaituo) [inline docs/ad/index.md:55]: Can we make it less certain of the sentence: "It's difficult for .."  

In https://github.com/opendistro-for-elasticsearch/anomaly-detection/issues/397, I said anomalies for an individual feature would need to be stronger/bigger if we use a multi-feature model than if a single-feature model.

Comment (kaituo) [reply docs/ad/index.md:55]: Also, can we mention the curse of dimensionality (https://en.wikipedia.org/wiki/Curse_of_dimensionality) somewhere?

Comment (ashwinkumar12345) [reply docs/ad/index.md]: Please review "A multi-feature model correlates anomalies across all its features. The [curse of dimensionality](https://en.wikipedia.org/wiki/Curse_of_dimensionality) makes it less likely for multi-feature models to identify smaller anomalies as compared to a single-feature model. Adding more features might negatively impact the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) of a model."

File: docs/ad/index.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ In this case, a feature is the field in your index that you to check for anomali
 
 For example, if you choose `min()`, the detector focuses on finding anomalies based on the minimum values of your feature. If you choose `average()`, the detector finds anomalies based on the average values of your feature.
 
-You can add a maximum of five features for a detector.
+A multi-feature model correlates anomalies across all its features. The [curse of dimensionality](https://en.wikipedia.org/wiki/Curse_of_dimensionality) makes it less likely for multi-feature models to identify smaller anomalies as compared to a single-feature model. Adding more features might negatively impact the [precision and recall](https://en.wikipedia.org/wiki/Precision_and_recall) of a model. A higher proportion of noise in your data might further amplify this negative impact. We recommend adding fewer features to your detector for a higher accuracy. By default, the maximum number of features for a detector is 5. You can adjust this limit with the `opendistro.anomaly_detection.max_anomaly_features` setting.
```
Comment (kaituo) [inline docs/ad/index.md:55]: The sentence "We recommend adding fewer features to your detector for a higher accuracy." is strong.  Can we tone it down and replace it with sth like "Selecting the optimal feature set is usually an iterative process.  It is a good idea to set up a historical detector to try different feature sets before moving to real-time detectors."

**Pull request merged**